supervisor/os/disk.py

```diff
@@ -0,0 +1,47 @@
+"""Home Assistant Operating-System DataDisk."""
```

Comment:
We might need other disk related files in the future.
Let's use `data_disk.py` for the filename to match the class here.